server/plugin.go

```diff
@@ -49,21 +51,38 @@ func (p *Plugin) OnActivate() error {
 	teamSplit := split[0]
 	channelSplit := split[1]
 
-	team, err := p.API.GetTeamByName(teamSplit)
-	if err != nil {
-		return err
+	team, appErr := p.API.GetTeamByName(teamSplit)
+	if appErr != nil {
+		return appErr
 	}
 	p.TeamID = team.Id
 
-	user, err := p.API.GetUserByUsername(p.configuration.Username)
+	botID, err := p.Helpers.EnsureBot(&model.Bot{
+		Username:    "aws-sns",
+		DisplayName: "AWS SNS Plugin",
+		Description: "A bot account created by the plugin AWS SNS",
+	},
+	plugin.ProfileImagePath("assets/profile.png")
```

Comment:
This will fix the build error

```suggestion
	plugin.ProfileImagePath("assets/profile.png"),
```